src/gen_module.sh

```diff
@@ -8,8 +8,9 @@ SCRIPT_DIR=$(cd $(dirname "$0"); pwd)
 }
 PYTHON_BIN=${PYTHON_BIN:-python}
 
-if [ $# -ne 6 ] && [ $# -ne 7 ]; then
-    echo "Usage: bash gen_module.sh <source-dir> <blender-dir> <target> <branch/tag/commit> <target-version> <output-dir> [<mod-version>]"
+if (( $# < 6 || 8 < $# )); then
+    echo "Usage: bash gen_module.sh <source-dir> <blender-dir> <target> <branch/tag/commit> <target-version> <output-dir> [<mod-version>] [<build-dir>]"
+    echo " if <build-dir> is specified then sphinx-in and generated_mods will be placed in <build-dir>"
```

Comment:
`build-dir` is an option for internal use.
Consider to use an environment variable.

Reply:
I removed `build-dir` option.